src/detect-engine-prefilter.h

```diff
@@ -1,4 +1,4 @@
-/* Copyright (C) 2016 Open Information Security Foundation
+/* Copyright (C) 2016-2020 Open Information Security Foundation
```

Comment:
nit: s/2020/2025

Reply:
see how long this branch as been going? ;)